documentation/dat/Skill.dat.md

```
@@ -0,0 +1,39 @@
+# Skill.dat
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)